courses/templates/courses/partials/after_the_course_body.html

```diff
@@ -71,7 +71,7 @@ <h2 class="after-course__stats-heading">
                 {{block.value.three_years_earnings_heading}}
             </h2>
 
-            {% if course_details.show_leo %}
+            {% if not course_details.show_leo %}
```

Comment:
Won't this stop the LEO stats being shown?

Reply:
I was rushing to fix it